dist/index.html

```diff
@@ -24,37 +24,62 @@
             <a href="#">News</a>
         </div>
     </div>
-    <div class="header-container">
-        <nav class="nav-wrapper">
-            <div class="mobile-nav">
-                <div class="hamburger">
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                    <span class="bar"></span>
-                </div>
-            </div>
-        </nav>
-        <div class="headline">
-            <h2>"Hello and Welcome!"</h2>
-            <h1>TECHEX </br> Blockchain-Expo 2023</h1>
-        </div>
-        <div class="headline-box">
-            <p>
-                This two day event will take place in November 2023, featuring 20
-                speakers and more than 500 participants. The theme for the two-day
-                Block-Chain Expo: Connecting the Blockchain Ecosystem
-            </p>
+    <div class="desktop-banner">
+        <div class="desktop-nav-banner">
+            <a href="#"><i class='fa fa-facebook'></i></a>
+            <a href="#"><i class="fa fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
         </div>
```

Comment:
## HOME AND ABOUT PAGE

- Hey, to enforce good use of the semantic tag, these are lists of items, so they should be wrapped in a `ul` tag and `li` to hold each list. let's kindly improve on this.

Reply:
changes  applied and fixed